dashboard/db/migrate/20160809170542_add_view_answers_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddViewAnswersUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
```

Comment:
Yep, better to avoid the three-state boolean problem when we really want there to be only two states: https://robots.thoughtbot.com/avoid-the-threestate-boolean-problem